CruCentralCoast/Events/EventsTVC.swift

```diff
@@ -0,0 +1,155 @@
+            if let indexPath = tableView.indexPathForRow(at: touchPoint) {
+                let cell = tableView.cellForRow(at: indexPath)
+                UIView.animate(withDuration: 0.2, animations: {
+                    cell?.transform = CGAffineTransform.identity
+                })
+            }
+        }
+    }
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return dataArray.count
+    }
+    
+    override func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 240
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueCell(EventsTableCell.self, indexPath: indexPath)
+        cell.event = dataArray[indexPath.item]
+        cell.selectionStyle = .none
+        return cell
+    }
+    
+//    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+//        guard let vc = UIStoryboard(name: "Events", bundle: nil).instantiateViewController(withIdentifier: "EventDetailsVC") as? EventDetailsVC else {
+//            assertionFailure("Probably used the wrong storyboard name or identifier here")
+//            return
+//        }
+//        vc.configure(with: self.dataArray[indexPath.item])
+//        vc.transitioningDelegate = self
+//        self.selectedCell = tableView.cellForRow(at: indexPath) as? EventsTableCell
+//        self.navigationController?.present(vc, animated: true, completion: nil)
+//    }
+}
+
+extension EventsTVC: UIViewControllerTransitioningDelegate {
+    
+    func animationController(forPresented presented: UIViewController, presenting: UIViewController, source: UIViewController) -> UIViewControllerAnimatedTransitioning? {
+        guard let selectedCell = self.selectedCell else {
+            return nil
+        }
+        guard let relativeFrame = selectedCell.cellMask.superview?.convert((selectedCell.cellMask.frame), to: nil) else {
+            return nil
+        }
+        self.statusBarIsHidden = true
+        self.originFrame = relativeFrame
+        let transition = EventDetailsTransition(originFrame: relativeFrame)
+        transition.presenting = true
+        return transition
+    }
+    
+    func animationController(forDismissed dismissed: UIViewController) -> UIViewControllerAnimatedTransitioning? {
+        self.statusBarIsHidden = false
+        let transition = EventDetailsTransition(originFrame: self.originFrame!)
+        transition.presenting = false
+        return transition
+    }
+}
+
+extension EventsTVC: UIGestureRecognizerDelegate {
+    func gestureRecognizer(gestureRecognizer: UIGestureRecognizer, shouldRecognizeSimultaneouslyWithGestureRecognizer otherGestureRecognizer: UIGestureRecognizer) -> Bool {
```

Comment:
Why is this needed? Might want to add a comment explaining it so someone doesn't accidentally remove it in future